apps/frontend/src/pages/leaderboard/[chipIssuer].tsx

```diff
@@ -37,6 +29,12 @@ const LeaderboardPage: React.FC = () => {
   useEffect(() => {
     const fetchInfo = async () => {
       if (typeof chipIssuer == "string") {
+        if (!Object.keys(communitiesEnum).includes(chipIssuer)) {
+          toast.error("Invalid chip issuer.");
+          router.push("/");
+          return;
+        }
+
         const chipIssuerMapping = communitiesEnum[chipIssuer];
         const chipIssuerEnum = ChipIssuerSchema.parse(chipIssuerMapping);
 
```

Comment:
There's a bug where if a user hasn't tapped anyone else, this leaderboard query errors out.

Reply:
Ah nice find, will address right now.